src/_ext.cpp

```diff
@@ -1703,6 +1705,11 @@ PYBIND11_MODULE(_ext, m) {
             jet_groomed_m.push_back(soft.m());
             jet_groomed_E.push_back(soft.E());
             jet_groomed_pz.push_back(soft.pz());
+
+            auto sd_struct = soft.structure_of<fastjet::contrib::SoftDrop>();
```

Comment:
@matthewfeickert and @rkansal47 I would like to hear suggestions on solving this failed test (https://github.com/scikit-hep/fastjet/actions/runs/10819914062/job/30018826074?pr=312#step:7:135). The ```RuntimeError std::bad_cast``` is coming from this specific line calling the dynamic casting in (https://github.com/alisw/fastjet/blob/master/fastjet/include/fastjet/PseudoJet.hh#L1126). The error thrown is happening on MacOS only. I can replicate this error on my own Mac but calling the same function in C++ only script with the same fastjet-contrib version did not give such an error. So I suspect this is related to pybind or how we compile the code. I am unfamiliar with both and would like to hear suggestions for things to try from any experts on this project.

Reply:
Probably want to ask (in addition to @rkansal47) other people that have touched the C++: @jpivarski @jmduarte @chrispap95 @lgray @aryan26roy